site/content/en/docs/user-manual/command-reference/transform.md

```diff
@@ -83,103 +83,164 @@ datum transform -t rename source-1 -- -e '|frame_(\d+)|\\1|'
 
 Annotation manipulations:
-- `remap_labels` - Renames, adds or removes labels in dataset
-- `project_labels` - Sets dataset labels to the requested sequence
-- `shapes_to_boxes` - Replaces spatial annotations with bounding boxes
-- `boxes_to_masks` - Converts bounding boxes to instance masks
-- `polygons_to_masks` - Converts polygons to instance masks
-- `masks_to_polygons` - Converts instance masks to polygons
-- `anns_to_labels` - Replaces annotations having labels with label annotations
-- `merge_instance_segments` - Merges grouped spatial annotations into a mask
-- `crop_covered_segments` - Removes occluded segments of covered masks
-- `bbox_value_decrement` - Subtracts 1 from bbox coordinates
+- [`remap_labels`](#remap_labels-transform) - Renames, adds or removes
+  labels in dataset
+- [`project_labels`](#project_labels-transform) - Sets dataset labels to
+  the requested sequence
+- [`shapes_to_boxes`](#shapes_to_boxes-transform) - Replaces spatial
+  annotations with bounding boxes
+- [`boxes_to_masks`](#boxes_to_masks-transform) - Converts bounding boxes
+  to instance masks
+- [`polygons_to_masks`](#polygons_to_masks-transform) - Converts polygons
+  to instance masks
+- [`masks_to_polygons`](#masks_to_polygons-transform) - Converts instance
+  masks to polygons
+- [`anns_to_labels`](#anns_to_labels-transform) - Replaces annotations having
+  labels with label annotations
+- [`merge_instance_segments`](#merge_instance_segments-transform) - Merges
+  grouped spatial annotations into a mask
+- [`crop_covered_segments`](#crop_covered_segments-transform) - Removes
+  occluded segments of covered masks
+- [`bbox_value_decrement`](#bbox_value_decrement-transform) - Subtracts
+  1 from bbox coordinates
+
+##### `rename` <a id="rename-transform"></a>
+
+Renames items in the dataset. Supports regular expressions.
+The first character in the expression is a delimiter for
+the pattern and replacement parts. Replacement part can also
+contain `string.format` tokens with `item` object available.
 
-Examples:
-
-- Split a dataset randomly to `train` and `test` subsets, ratio is 2:1
+Usage:
 ``` bash
-datum transform -t random_split -- --subset train:.67 --subset test:.33
+rename [-h] [-e REGEX]
 ```
 
-- Split a dataset for a specific task. The tasks supported are
-classification, detection, segmentation and re-identification.
+Optional arguments:
+- `-h`, `--help` (flag) - Show this help message and exit
+- `-e`, `--regex` (string) - Regex for renaming in the form
+  `<sep><search><sep><replacement><sep>`
 
-``` bash
-datum transform -t split -- \
-  -t classification --subset train:.5 --subset val:.2 --subset test:.3
+Examples:
+Replace 'pattern' with 'replacement':
+```bash
+datum transform -t rename -- -e '|pattern|replacement|'
+```
 
-datum transform -t split -- \
-  -t detection --subset train:.5 --subset val:.2 --subset test:.3
+Remove the `frame_` prefix from item ids:
+```bash
+datum transform -t rename -- -e '|frame_(\d+)|\1|'
```

Comment:
If the goal is to remove a prefix, wouldn't it be simpler to do `|^frame_||`?

Reply:
Updated

Reply:
The replacement part is now `\\1` - that seems wrong.

Reply:
Fixed